docs/source/how_to/how_to_derivatives.ipynb

```diff
@@ -0,0 +1,276 @@
+{
```

Comment:
<p>Maybe I'd define <code>dict_params</code> at the very top. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/optimagic/pull/512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>